keyman_trust_stub.go

```diff
@@ -13,10 +13,6 @@ func DeleteTrustedRootByName(commonName string, prompt string) error {
 // AddAsTrustedRoot adds the certificate to the user's trust store as a trusted
 // root CA. If prompt is provided, privilege escalation will be requested (if
 // required) and the user will be prompted with the given text.
-func (cert *Certificate) AddAsTrustedRoot(prompt string) error {
-	return fmt.Errorf("AddToUserTrustStore is not supported on this platform")
-}
-
-func (cert *Certificate) IsInstalled() (bool, error) {
-	return false, fmt.Errorf("IsInstalled is not supported on this platform")
+func (cert *Certificate) AddAsTrustedRootIfNeeded(prompt string) error {
```

Comment:
This signature doesn't match the other platforms.

Reply:
geez. thanks. fixing